Style Guide fixes, typos, and fixed links (#4437)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/analytics_overview.md

{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}

#### Numbers {#integers}

> Numeric attributes have a wide variety of use cases. Incrementing number custom attributes are useful for storing the number of times a given action or event has occurred. Standard numbers have all sorts of usages, for example: recording shoe size, waist size, or number of times a user has viewed a certain product feature or category.

Reviewer: ```suggestion
Numeric attributes have a wide variety of use cases. Incrementing number custom attributes are useful for storing the number of times a given action or event has occurred. Standard numbers have all sorts of usages, such as recording shoe size, waist size, or the number of times a user has viewed a certain product feature or category.
```

{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}

#### Booleans (true/false)

> Boolean attributes are useful for storing subscription statuses and other simple binary data about your users. The input options that we provide allow you to find users that have explicitly had a variable set to a true/false value in addition to those that don't have any record of that attribute recorded yet.

Reviewer: ```suggestion
Boolean attributes are useful for storing subscription statuses and other simple binary data about your users. The input options that we provide allow you to find users that have explicitly had a variable set to a boolean in addition to those that don't have any record of that attribute recorded yet.
```

>  If you would like to segment on the number of times a specific purchase has occurred, you should also record that purchase individually as an [incrementing custom attribute][12].

## Taxi/ride-sharing app use case {#example-case}

> For this example case, let's consider a taxi/ride-sharing app (such as Hailo or Lyft) that wants to decide what user data to collect. The following questions and brainstorming process are a great model for marketing and development teams to follow. By the end of this exercise, both teams should have a solid understanding of what custom events and attributes make sense to collect in order to help meet their goal.

Reviewer: ```suggestion
For this example, let's consider a ride-sharing app that wants to decide what user data to collect. The following questions and brainstorming process are a great model for marketing and development teams to follow. By the end of this exercise, both teams should have a solid understanding of what custom events and attributes make sense to collect in order to help meet their goal.
```
